main/autogen/autoloader.cc

```diff
@@ -222,6 +223,14 @@ string DefTree::renderAutoloadSrc(const core::GlobalState &gs, const AutoloaderC
             }
             fmt::format_to(std::back_inserter(buf), "}})\n", fullName);
         }
+
+        if (pkgName.exists()) {
+            ENFORCE(!gs.packageDB().empty());
+            const string_view shortName = pkgName.shortName(gs);
+            const string_view mungedName = shortName.substr(0, shortName.rfind(core::PACKAGE_SUFFIX));
```

Comment:
```suggestion
            const string_view mungedName = shortName.substr(0, shorName.size() - core::PACKAGE_SUFFIX.size());
```
Nit: does this need a find?

Reply:
No, will work with backwards index math as well.